src/Features/Core/Portable/ExtractMethod/MethodExtractor.TriviaResult.cs

```diff
@@ -40,16 +40,12 @@ public async Task<OperationStatus<SemanticDocument>> ApplyAsync(GeneratedCode ge
 
                 var annotationResolver = GetAnnotationResolver(callsite, method);
                 var triviaResolver = GetTriviaResolver(method);
-                if (annotationResolver == null || triviaResolver == null)
-                {
-                    // bug # 6644
-                    // this could happen in malformed code. return as it was.
-                    var status = new OperationStatus(OperationStatusFlag.None, FeaturesResources.can_t_not_construct_final_tree);
-                    return status.With(document);
-                }
 
-                return OperationStatus.Succeeded.With(
-                    await document.WithSyntaxRootAsync(_result.RestoreTrivia(root, annotationResolver, triviaResolver), cancellationToken).ConfigureAwait(false));
+                // Failed to restore the trivia.  Just return whatever best effort result is that we have so far.
+                return annotationResolver == null || triviaResolver == null
+                    ? OperationStatus.Succeeded.With(document)
+                    : OperationStatus.Succeeded.With(
```

Comment:
can't find anything that actually hits this, and it forces tons of tree work up front.  Changing this to effectively say: we'll still allow the operation to proceed, even if we have an issue with trivia fixup at teh end.